apps/prairielearn/src/pages/authCallbackLti/authCallbackLti.js

```diff
@@ -131,7 +132,7 @@ router.post('/', function (req, res, next) {
         res.cookie('pl_authn', pl_authn, {
           maxAge: config.authnCookieMaxAgeMilliseconds,
           httpOnly: true,
-          secure: true,
+          secure: shouldSecureCookie(req),
```

Comment:
I'm wondering if we should have (or why we didn't) push the lib/authn library into the LTI 1.1 cookie, or use this as the reason to clean it up?

Reply:
PRs welcome if you think that would be a valuable change!